client/scripts/models/NotificationSubscription.ts

```diff
@@ -6,17 +6,23 @@ class NotificationSubscription {
   public readonly objectId: string;
   public readonly createdAt: moment.Moment;
 
+  private _immediateEmail: boolean;
+  public get immediateEmail() { return this._immediateEmail; }
+  public enableImmediateEmail() { this._immediateEmail = true; }
+  public disableImmediateEmail() { this._immediateEmail = false; }
```

Comment:
"This seems a bit weird to directly have this boolean here, even in the short term. We'll be extending this with the other options (daily, weekly, monthly, etc etc), might as well make the change now." Extends from the choice of the Subscription Model